src/testIntegration/resources/application-test-integration.yml

```diff
@@ -44,10 +8,10 @@ offender.retention:
     ap.check.enabled: false
 
 notify.enabled: false
-digital.prison.service.url: https://digital-dev.prison.service.justice.gov.uk
 
 deletion.grant.enabled: true
 offender.deletion.review.required: false
 image.recognition.deletion.enabled: true
 
-server.shutdown: immediate
+hmpps.sqs:
+  provider:
```

Comment:
Not sure what this is doing - is it required?

Reply:
Yes it is, I believe the sqs plugin provides `aws` as the default. I have put that in there to override the default.

I have a few messaging services that I do not want be created in these tests that have:

@ConditionalOnExpression("{'aws', 'localstack'}.contains('${hmpps.sqs.provider}')")

Reply:
Hmmm that's a bit rubbish, it should default to no-op. OK cool.